dcpy/configuration.py

```diff
@@ -0,0 +1,16 @@
+from os import environ as env
+
+_DEV_FLAG = env.get("DEV_FLAG")
+if _DEV_FLAG and _DEV_FLAG not in ("true", "false"):
+    raise ValueError(f"Invalid value for env var 'DEV_FLAG': '{_DEV_FLAG}'")
+DEV_FLAG = _DEV_FLAG == "true"
+
+CI = "CI" in env
+
+BUILD_NAME = env.get("BUILD_NAME")
+if DEV_FLAG and not BUILD_NAME:
+    raise ValueError("'BUILD_NAME' env var needed with 'DEV_FLAG'")
+
+RECIPES_BUCKET = "edm-recipes"
+PUBLISHING_BUCKET = "edm-publishing"
+DEV_BUCKET = f"de-dev-{BUILD_NAME}" if DEV_FLAG else None
```

Comment:
Maybe this `DEV_BUCKET` should just be the env var instead of `DEV_FLAG`. Would allow flexibility without adding an additional env var to check. But currently, it's nice just having a checkbox in a GHA

Reply:
yea a boolean like `DEV_FLAG` at the top-level to drive downstream variables seems nice